Octokit.Tests/Helpers/StringExtensionsTests.cs

```diff
@@ -57,9 +57,11 @@ public class TheExpandUriTemplateMethod
             [InlineData("https://host.com/path?name=other", "https://host.com/path?name=other")]
             [InlineData("https://host.com/path?name=example name.txt", "https://host.com/path{?name}")]
             [InlineData("https://host.com/path", "https://host.com/path{?other}")]
+            [InlineData("https://host.com/path?name=example name.txt,label=labeltext", "https://host.com/path{?name,label}")]
+            [InlineData("https://host.com/path?name=example name.txt,label=labeltext", "https://host.com/path{?name,label,other}")]
```

Comment:
Is that the proper expansion? Or should it be `?name=example name.txt&label=labeltext`